src/device/usbd.c

```diff
@@ -59,6 +61,8 @@ typedef struct {
   uint8_t ep2drv[8][2];    // map endpoint to driver ( 0xff is invalid )
 }usbd_device_t;
 
+const uint8_t INVALID_DRV_TOKEN  = 0xFFu;

```

Comment:
This makes it clear what the meaning of 0xFF is, but can be removed if deemed too verbose.

Reply:
yeah, I like it, magic number is always bad, I was tentative to use 0xff or (-1) previously. So I did a hardcode number first, which is not really good. After reviewing it, 0xff is better since we don't run into complement 2 implementation of -1 on weird platform.